### ISSUE TITLE
Failed: bind filles or directories from project service repo to the container

### ISSUE THREAD

**Issue by petrus-v** [mlfmonde/cluster#10]
There are two mains problem to bind files or directories from the project into the container:

1. as docker-compose is run in consul container, but container launch on the host machine, past MUST be the same so I suggest to move ``./deploy`` directory to ``/deploy``

2. we needs to find a way to avoid or escape columns ":" in directory path  

ERROR: Volume '/deploy/service3ca8a@2018-03-29T23:00:29.139095/myconfig.file:/etc/service/config.ini has incorrect format, should be external:internal[:mode]

Or report and contribute that issue to docker-compose

**Comment by petrus-v** [comment]
Suggest "2. we needs to find a way to avoid or escape columns ":" in directory path" appear complicated after few investigation even docker do not support it:

```bash
root@oak:~/toto:tata:titi# docker run -v /root/toto\:tata\:titi/odoo/:/tmp/odoo alpine cat /tmp/odoo/f
docker: Error response from daemon: invalid volume specification: '/root/toto:tata:titi/odoo/:/tmp/odoo'.
```

btw, the example to reproduce where working directory is ``root/toto\:tata\:titi/``:
```bash
cat docker-compose.yml 
version: '3.0'
services:

  odoo:
    image: nginx:alpine
    volumes:
      - ./odoo:/tmp/odoo

```

**Comment by petrus-v** [comment]
This already fixed and merged through PR #11
